readme/Readme.scalatex

```diff
@@ -0,0 +1,324 @@
+@import Main._
+@import scalatex.site._
+
+@a(
+  href:="https://github.com/scalalandio/chimney",
+  position.absolute,
+  top:=0,right:=0,border:=0,
+  img(
+    src:="https://s3.amazonaws.com/github/ribbons/forkme_right_orange_ff7600.png",
+    alt:="Fork me on GitHub"
+  )
+)
+
+@sect("Chimney", "Scala library for boilerplate-free data transformations")
+  @sect{What Chimney does}
+    @p
+      In strongly-typed language programmer's daily life sometimes happens
+      to transform an object of one type to another object which contains
+      number of the same or similar fields in their definitions.
+    @hl.scala
+      case class MakeCoffee(id: Int, kind: String, addict: String)
+      case class CoffeeMade(id: Int, kind: String, forAddict: String, at: ZonedDateTime)
+    @p
+      Usual approach is to just rewrite fields one by one
+    @hl.scala
+      val command = MakeCoffee(id = Random.nextInt,
+                               kind = "Espresso",
+                               addict = "Piotr")
+      val event = CoffeeMade(id = command.id,
+                             kind = command.kind,
+                             forAddict = command.addict,
+                             at = ZonedDateTime.now)
+    @p
+      While the example stays lean, in real-life code we usually end up with tons
+      of such boilerplate, especially when:
+      @ul
+        @li
+          we maintain typed schema and want to migrate between multiple schema
+          versions
+        @li
+          we apply practices like DDD (Domain-Driven-Design) where suggested
+          approach is to separate model schemas of different bounded contexts
+        @li
+          we use code-generation tools like Protocol Buffers that generate primitive
+          types like @hl.scala{Int} or @hl.scala{String}, while you'd prefer to
+          use value objects in you domain-level code to improve type-safety
+          and readability
+    @p
+      Chimney uses type-level meta-programming techniques based on
+      @lnk("Shapeless", "https://github.com/milessabin/shapeless") library
+      and provides compact DSL on top of transformation rules derivation engine
+      to transform your objects with as little boilerplate as possible
+      and compile-time safety!
```

Comment:
Chimney provides a compact DSL with which you can define transformation rules and transform your objects with as little boilerplate as possible. Underneath it uses type-level meta-programming based on Shapeless and type-class derivation to give you type-safety at compile-time!